hier_config/models.py

```diff
@@ -100,6 +100,7 @@ class Platform(str, Enum):
     CISCO_IOS = auto()
     CISCO_NXOS = auto()
     CISCO_XR = auto()
+    FORTIGATE_FORTIOS = auto()
```

Comment:
Should this be `FORTINET_FORTIOS` in order to follow the ≤COMPANY≥_<OS NAME> convention?